qa/tasks/s3tests.py

```diff
@@ -461,27 +543,81 @@ def task(ctx, config):
 
     log.debug('s3tests config is %s', config)
 
+    global sts_variable
```

Comment:
I just learned that you can share a variable through different tasks via `ctx` so you could just declare `sts_variable` in another task, you can declare `ctx.sts_variable=true` in say `keycloak.py`.